### ISSUE TITLE
Report severity appears to be ignored

### ISSUE THREAD

**Issue by pdxwolfy** [leny/atom-w3c-validation#29]
For instance, this code:

``` css
fieldset {
  background-color: #ececec;
}
```

yields a warning that "You have no color set (or color is set to transparent) but you have set a background-color. Make sure that cascading of colors keeps the text reasonably legible." regardless of whether the CSS Report Severity is set to All or Normal. [Just checked: it also warns when the severity is set to Most Important or No warnings.] 

On the W3C site, a severity of Normal does not display this warning.

Using 0.4.0 with Atom 1.10.2 on Mac OS X 10.11.6

**Comment by leny** [comment]
Thanks.
I will try to take a look on this current the week.

**Comment by pdxwolfy** [comment]
Leny, I'd be happy to try to take a look at this. Would you be willing to add me as a collaborator?

**Comment by JanisE** [comment]
Same here – set to "No warnings", and warnings still appear.

**Comment by usizu** [comment]
any chance this will get fixed?

**Comment by leny** [comment]
Hi,

I'm sorry, I'm not using Atom since years now. 

I will be happy to merge any pull request, or give the lead on this plugin to anyone who wants to handle his maintenance.
